stable_baselines3/a2c/a2c.py

```diff
@@ -114,6 +114,12 @@ def __init__(
         if _init_setup_model:
             self._setup_model()
 
+    policy_aliases: Dict[str, Type[BasePolicy]] = {
```

Comment:
is there a reason for defining it after the `_init_` and not before?

Reply:
None, I hesitated
Would defining it before `__init__` imply defining it before the class's docstring ?

Reply:
I would actually define it right after the class docstring (if possible)

Reply:
Done